rednotebook/info.py

```diff
@@ -39,6 +39,7 @@ def _(string):
 bug_url = "https://github.com/jendrikseipp/rednotebook/issues"
 version_url = "https://raw.githubusercontent.com/jendrikseipp/rednotebook/master/rednotebook/info.py"
 contributors_url = "https://github.com/jendrikseipp/rednotebook/graphs/contributors"
+discussion_url = "https://github.com/jendrikseipp/rednotebook/discussions"
```

Comment:
I think that it would be better to redirect the user to `.../discussions/new/?category=ideas`. @jendrikseipp  what do you think?